src/pages/Landing/index.tsx

```diff
@@ -47,8 +47,10 @@ export default () => {
 
   const onSubmit = (e: FormEvent<Element>) => {
     e.preventDefault();
+    addNewCustomer(newUser as Customer)
     setNewUser({ isCool: false });
     setIsModalOpen(false);
+    queryClient.invalidateQueries('CustomerQuery');
```

Comment:
as we chatted on irc, the query name is incorrect, it needs to match the query name that is used when fetching the customers

Reply:
The key/name of the query you are invalidating still isn't quite right, as you pointed out on slack, the query here: https://github.com/content-services/React-typescript-demo/blob/main/src/pages/Landing/index.tsx#L40-L44 is using 'customers' as the key